reana.yaml

```diff
@@ -7,9 +7,6 @@ inputs:
     events: 20000
     data: results/data.root
     plot: results/plot.png
-environments:
```

Comment:
Can you please amend also `reana-cwl.yaml` and `reana-yadage.yaml`?  And also update `README.rst` accordingly?

```console
$ git grep -Hni 'environments:' 
README.rst:146:    environments:
reana-cwl.yaml:5:environments:
reana-yadage.yaml:7:environments:
reana.yaml:10:environments:
```